console/src/components/Services/Data/Common/Components/ReloadEnumValuesButton.js

```diff
@@ -1,32 +1,17 @@
 import React from 'react';
-import ReloadEnumMetadata from '../../../Settings/MetadataOptions/ReloadMetadata';
-import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
-import Tooltip from 'react-bootstrap/lib/Tooltip';
-import styles from '../../../../Common/Common.scss';
+import ReloadMetadata from '../../../Settings/MetadataOptions/ReloadMetadata';
 
-const ReloadEnumValuesButton = ({ isEnum, dispatch, tooltipStyle }) => {
-  if (!isEnum) return null;
+const TOOLTIP_TEXT =
+  'Reload enum values in your GraphQL schema after inserting, updating or deleting enum values';
 
-  const tooltip = (
-    <Tooltip id="tooltip-reload-enum-metadata">
-      Reload enum values in your GraphQL schema after inserting, updating or
-      deleting enum values
-    </Tooltip>
+const ReloadEnumValuesButton = ({ isEnum, dispatch }) =>
+  isEnum && (
+    <ReloadMetadata
+      buttonText="Reload enum values"
+      buttonToolTip={TOOLTIP_TEXT}
+      checkBoxVisible={false}
```

Comment:
Someone's first thought might be _What checkbox?_ Let's make it more descriptive. 

Couple suggestions:
- `showReloadRemoteSchemas`
- `canReloadRemoteSchemas`
- `reloadRemoteSchemasButton`